src/server/controllers/userController.js

```diff
@@ -0,0 +1,54 @@
+const client = require('../models/cipherModel');
+
+const userController = {};
+
+userController.getAllUsers = (req, res, next) => {
+  client.query('SELECT * FROM users', (err, results) => {
+    res.json(results.rows);
+  });
+};
+
+userController.createUser = (req, res, next) => {
+  const {
+    username, password, firstname, lastname, email,
+  } = req.body;
+  console.log(`I signed up ${username}, ${password}, ${firstname}, ${lastname}, ${email}`);
```

Comment:
Was this intentional? Logging username and password? If not let's remove it

Reply:
Yes for testing purposes, its gone